src/express/middleware/authenticate.ts

```diff
@@ -12,8 +12,8 @@ export default (config: SanitizedConfig): PayloadAuthenticate => {
       const collectionMethods = [...enabledMethods];
 
       if (Array.isArray(collection.auth.strategies)) {
-        collection.auth.strategies.forEach(({ strategy }) => {
-          collectionMethods.unshift(strategy.name);
+        collection.auth.strategies.forEach(({ name }) => {
```

Comment:
Same as above...

Reply:
This is a bit trickier. This is actually the reason why I've added the name. If we use the strategy object to get the name, and we're using the factory function, that means we might be initializing the strategy object twice unnecessarily. Meaning that would be called both here and on collection init. Thoughts?